management/server/file_store.go

```diff
@@ -163,6 +230,25 @@ func (s *FileStore) SaveAccount(account *Account) error {
 	s.mux.Lock()
 	defer s.mux.Unlock()
 
+	if len(account.Groups) == 0 {
```

Comment:
I think it makes sense to move the logic of adding the default group to the AccountManager since it is not a business of the FIleStore